make/migrations/postgresql/0030_2.0.0_schema.up.sql

```diff
@@ -1,4 +1,8 @@
 ALTER TABLE admin_job ADD COLUMN job_parameters varchar(255) Default '';
+
+/*record the data version to decide whether the data migration should be skipped*/
```

Comment:
replease `data_version` with `cur_version`? it's easy to understand.

Reply:
What does the `cur_version` stand for? Is it current version?
The table `schema_migrations` contains a column `version` which indicates the schema version. Use `data_version` to distinguish it
```
CREATE TABLE schema_migrations(version bigint not null primary key, dirty boolean not null);
```